tests/test_network.py

```diff
@@ -39,6 +42,22 @@ def test_add_data_sample(factory, session):
     session.add_data_sample(spec)
 
 
+def test_add_data_sample_located_in_shared_path(factory, session, node_cfg):
+    spec = factory.create_dataset()
+    dataset = session.add_dataset(spec)
+
+    spec = factory.create_data_sample(test_only=True, datasets=[dataset])
+
+    # move data sample to substra backend shared file system
+    dst = node_cfg.shared_path
+    dst = dst if dst.endswith('/') else dst + '/'
+    shutil.move(spec.path, dst)
```

Comment:
While running the tests with docker compose, this line fails with the following error:
`E               shutil.Error: Destination path '/tmp/substra/data-0' already exists`

Reply:
Good catch, the tests suite can run only once :)

Reply:
Done

Reply:
Data samples are now moved to a unique directory:
```
/tmp/peer-1
├── 2r4bixf2
│   └── data-0
│       └── data.csv
├── 63s3g6ab
│   └── data-0
│       └── data.csv
├── ax5vulhq
│   └── data-0
│       └── data.csv
```